editoast/src/modelsv2/railjson.rs

```diff
@@ -57,27 +38,88 @@ pub async fn persist_railjson(
         extended_switch_types,
         neutral_sections,
     } = railjson;
+
     if version != RAILJSON_VERSION {
         return Err(RailJsonError::UnsupportedVersion {
             actual: version,
             expected: RAILJSON_VERSION.to_string(),
         }
         .into());
     }
-    futures::try_join!(
-        persist!(TrackSectionModel, track_sections),
-        persist!(BufferStopModel, buffer_stops),
-        persist!(ElectrificationModel, electrifications),
-        persist!(DetectorModel, detectors),
-        persist!(OperationalPointModel, operational_points),
-        persist!(RouteModel, routes),
-        persist!(SignalModel, signals),
-        persist!(SwitchModel, switches),
-        persist!(SpeedSectionModel, speed_sections),
-        persist!(SwitchTypeModel, extended_switch_types),
-        persist!(NeutralSectionModel, neutral_sections),
-    )
-    .map(|_| ())
+
+    connection
```

Comment:
Removing the `futures::try_join!()` in favor of this big block should have a performance impact if I'm not wrong. `try_join!()` does launch all the futures concurrently, but I do believe this new code will run sequentially.

Reply:
OK, after discussion, we acknowledge that it might be a performance regression, but will avoid starving the DB pool which might kill the entire service, it's a balance ⚖️

Reply:
It could be interesting to measure how much the performance is degraded. As far as I know, this function is either used for:

1. tests, where no big infra is imported -- so it's irrelevant
2. importing an infra using the frontend, we're however limited by the size of request payloads, which kinda breaks the whole feature (which should be removed until we find a solution imo)
3. loading a recently imported infra while testing dags or creating a dump, in which case a small perf degradation is painless, but a big one can be pretty impactful on the workflow

If performance of `persist` becomes problematic, we could make it accept an `Either<&mut conn, db_pool>` and make the queries concurrent if the pool is provided. We could then provide the pool for the CLI import, but keep the single connection for the endpoint to avoid starvation.

Reply:
Well, the problem with having a `pool` directly is that we might starve the pool... and therefore, `/health` which does check the DB is up and running won't have an available connection, then take more than a second to respond. If the deployment is configured to restart the service when `/health` is too slow (which is our case), then it starts to be a bigger problem.